microsoft-edge/progressive-web-apps-chromium/ux.md

```diff
@@ -0,0 +1,112 @@
+
+Since PWAs are deeply integrated in Windows, they show up anywhere other types of Windows apps get displayed, including in the **Apps & features** system setting.
+
+:::image type="complex" source="./media/pwa-in-apps-and-features-settings.png" alt-text="The list of installed apps on Windows also display PWAs" lightbox="./media/pwa-in-apps-and-features-settings.png":::
+   The list of installed apps on Windows also display PWAs
+:::image-end:::
+
+
+## Windows integration
+
+PWAs are displayed like native apps on Windows. They appear in the Taskbar (where they can be pinned), in the Start Menu, or when switching between apps with `Alt`+`Tab`.
+
+:::image type="complex" source="./media/windows-start-menu.png" alt-text="PWAs and native apps side by side in the Taskbar and Start Menu" lightbox="./media/windows-start-menu.png":::
+   PWAs and native apps side by side in the Taskbar and Start Menu
+:::image-end:::
+
+PWAs can also expose common tasks to users as shortcuts that appear in the app's icon contextual menu (right-click). Learn more about [defining shortcuts][AppShortcutsFeature].
+
+:::image type="complex" source="./media/pwa-shortcuts-in-taskbar.png" alt-text="Common tasks are listed in the Taskbar contextual menu" lightbox="./media/pwa-shortcuts-in-taskbar.png":::
+   Common tasks are listed in the Taskbar contextual menu
+:::image-end:::
+
+PWAs can also display notifications in the operating system's own notification service. This helps users re-engage with your app. Learn more about [using notifications, push messages, and badges][PushNotficationsFeature].
+
+:::image type="complex" source="./media/windows-action-center.png" alt-text="Notifications in Windows Action Center" lightbox="./media/windows-action-center.png":::
+   Notifications in Windows Action Center
+:::image-end:::
+
+### Starting PWAs when the user logs in
+
+PWAs can launch automatically when the user logs in on Windows so they can immediately engage with the app.
+
+To turn this feature on for an installed PWA:
+
+1.  Open Microsoft Edge.
+1.  Navigate to `edge://apps`.
+1.  Open the contextual menu (right-click) on the app in the list of installed app.
+1.  Select **Start app when you sign in**.
+
+:::image type="complex" source="./media/turn-on-run-on-os-login-flag.png" alt-text="Use the contextual menu to turn on the Start app when you sign in feature in Microsoft Edge" lightbox="./media/turn-on-run-on-os-login-flag.png":::
+   Use the contextual menu to turn on the **Start app when you sign in** feature in Microsoft Edge
+:::image-end:::
+
+Upon installation of an app, users are also given a chance to turn on the feature in the post-installation dialog.
+
+:::image type="complex" source="./media/post-install-run-on-os-login.png" alt-text="The post-installation dialog automatically opens once an app is installed" lightbox="./media/post-install-run-on-os-login.png":::
```

Comment:
resize png smaller

Reply:
Yeah, I've been annoyed by this, but I'm not really sure what to do here. The most problematic images are those that are tall. The site tries to make most images as wide as the text content. And because of this, the images that are quite tall end up needing almost 1 full screen to display. I agree this makes it hard to follow.
What would you advise in this case? Let's take the specific example of the start menu image here:

microsoft-edge\progressive-web-apps-chromium\media\windows-start-menu.png
https://review.docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/media/windows-start-menu.png?branch=pr-en-us-1429#lightbox

It feels odd to cut it in the middle and only keep the bottom part.
Is there a way to tell the site to only show smaller thumbnails and let users click on them to view the images full size?

Reply:
We need to research in the DCG and on the web, triple-colon syntax.  I heard rumors of a way to control image size (without altering the .png file).  In this particular case, I don't think there's a need for a lightbox that's actually bigger than the initial size.
Side question: why use a lightbox in those cases where the lightbox result is the same size as the original image in the context of the article/page?
I suggest for this particular image, resizing to maybe 70%; altering the .png file by using the Resize UI in Paint.  That might cause the lightbox to be no bigger than the orig page context, which is not a proper lightbox behavior.